components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/utils/mappings/WorkflowMappingUtil.java

```diff
@@ -74,6 +74,8 @@ public static WorkflowInfoDTO fromWorkflowsToInfoDTO(Workflow workflow) {
             workflowInfoDTO.setWorkflowType(WorkflowInfoDTO.WorkflowTypeEnum.APPLICATION_REGISTRATION_SANDBOX);
         } else if (workflow.getWorkflowType().equals("AM_API_STATE")) {
             workflowInfoDTO.setWorkflowType(WorkflowInfoDTO.WorkflowTypeEnum.API_STATE);
+        } else if (workflow.getWorkflowType().equals("AM_API_PRODUCT_STATE")) {
```

Comment:
This can have an NPE right?

Reply:
This is used in retrieving the workflow information from database. Since `WF_TYPE` cannot be null in database, due to [1], there won't be NPE here.

[1] https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql#L1616